### PR TITLE
Post on how we use Tettra

### PR DESCRIPTION
**Pull request by ABizzinotto**
**IMPORTANT: blog posts must be approved by two reviewers before they can be published, one of which must be a native English speaker.**

[Add any information about this post here. Leave the note above when opening the Pull Request.]

This post links to the one about knowledge bases.

### CODE REVIEW

File: _posts/2020-09-17-using-tettra.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+To keep our content organized, we use Tettra's categories and folders structure. Each category represents an area and each folder a division within that area. For example, under the Sales category, we have the Pre-Sales and Marketing folders.
+
+We also use Folders to keep our Draft pages before they are officially published to the whole team.
+
+## Adding Content
+
+Tettra allows you to create pages inside the tool as well as add a link to external documents (Google Doc or Google Sheet). This lets us create new pages as needed as well as link to our documents available through Google Docs and Google Sheets, keeping all the information in a single place.
+
+To make sure we keep things organized and standardized, we created a style guide for our Tettra pages that can be found under the Using Tettra category. Additionally. we also have a standard procedure for adding content.
```
Comment (lubc) [inline _posts/2020-09-17-using-tettra.md:32]:
```suggestion
To make sure we keep things organized and standardized, we created a style guide for our Tettra pages that can be found under the Using Tettra category. Additionally. We also have a standard procedure for adding content.
```

File: _posts/2020-09-17-using-tettra.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+However, sometimes a team member wants to see something in the knowledge base but is not sure what's the best way to add it. In that case, they can create a suggestion and either assign it to the best team member to write a page or to the knowledge base admin, who'll make sure the suggestion is addressed.
+
+This way we make sure all content added to the knowledge base is accurate and we also make sure we address all concerns from the team and add all the content they need there.
+
+## Adding a Category
+
+In order to make sure our knowledge base doesn't get out of order with too many categories and folders, we have a procedure in place to add new categories.
+
+Whenenver a team member wants to add a new category to the knowledge base or remove a category from it, they can create a suggestion and assign it to the knowledge base admin, who'll make sure the suggestion is addressed.
```
Comment (lubc) [inline _posts/2020-09-17-using-tettra.md:46]:
```suggestion
Whenever a team member wants to add a new category to the knowledge base or remove a category from it, they can create a suggestion and assign it to the knowledge base admin, who'll make sure the suggestion is addressed.
```

File: _posts/2020-09-17-using-tettra.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Using Tettra to Keep our Content Organized"
+date: 2020-09-17 11:00:00
+reviewed: 2020-09-17 11:00:00
+categories: ["team", "remote"]
+author: "abizzinotto"
+---
+
+At OmbuLabs we use [Tettra](https://app.tettra.co) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](link to kb article), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
```
Comment (lubc) [inline _posts/2020-09-17-using-tettra.md:10]: The landing page has probably more details
```suggestion
At OmbuLabs we use [Tettra](https://tettra.com/) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](link to kb article), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
```

Comment (lubc) [reply _posts/2020-09-17-using-tettra.md:10]: Also, you're missing a link in: `[here](link to kb article)`

**Review by lubc — COMMENTED** [review]
@ABizzinotto Great article!

I think it would be nice to also put some screenshots in the article. What do you think?

File: _posts/2020-09-17-using-tettra.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+In order to keep those synced, we created a custom integration via Tettra's API. This way, all the information available in our blog is also accessible from the knowledge base.
+
+## Keeping the Knowledge Base Up to Date
+
+Documentation becomes obsolete, policies change and so on. Adding content is a good way to make sure all the information is safely stored, but removing obsolete content and updating existing pages is also important to keep the knowledge base clean and relevant.
+
+In order to ensure our content is always up to date, we use Tettra's inbuilt feature of notifying the admin every time a page has gone three months without an update. The admin gets a notification and can check the page in question and update it if needed or tag the best team member to review and update.
+
+Having a notification in Slack is a great way to ensure the admin knows when to review pages and which pages to review. And having the clock running for each page individually ensures we don't have to dedicate a lot of time to review the entire knowledge base ever so often.
```
Comment (FionaDL) [inline _posts/2020-09-17-using-tettra.md:78]:
```suggestion
Having a notification in Slack is a great way to ensure the admin knows when to review pages and which pages to review. And having the clock running for each page individually ensures we don't have to dedicate a lot of time to reviewing the entire knowledge base every so often.
```

File: _posts/2020-09-17-using-tettra.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+## Conclusion
```
Comment (FionaDL) [inline _posts/2020-09-17-using-tettra.md:80]: There is a lot of "easy" in the last paragraph...consider:
It integrates nicely with the tools we use, has an easy to use interface and allows us to  keep the content updated and accessible.

**Review by FionaDL — COMMENTED** [review]
@ABizzinotto looks good! A few comments :)

**Comment by ABizzinotto** [comment]
> @ABizzinotto Great article!
> 
> I think it would be nice to also put some screenshots in the article. What do you think?

@lubc I'm all for screenshots hehe but not sure where I'd add them though. Thoughts?

**Comment by lubc** [comment]
> @lubc I'm all for screenshots hehe but not sure where I'd add them though. Thoughts?

@ABizzinotto maybe you can put one screenshot in some of the sections that you cover, like categories or slack integration.

**Comment by ABizzinotto** [comment]
@lubc done :)

File: _posts/2020-09-17-using-tettra.md (hunk 3 above)
Comment (lubc) [reply _posts/2020-09-17-using-tettra.md]: > Also, you're missing a link in: `[here](link to kb article)`

@ABizzinotto ping

we should remove that until the other article is published.

File: _posts/2020-09-17-using-tettra.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Using Tettra to Keep our Content Organized"
+date: 2020-09-17 11:00:00
+reviewed: 2020-09-17 11:00:00
+categories: ["team", "remote"]
+author: "abizzinotto"
+---
+
+At OmbuLabs we use [Tettra](https://tettra.com/) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](https://www.ombulabs.com/blog/team/agile/remote/organizing-team-knowledge-bases.html), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
```
Comment (etagwerker) [inline _posts/2020-09-17-using-tettra.md:10]: @ABizzinotto Maybe you can explain a little bit more about our motivation behind this? I think many remote teams can relate to this: Slack is a great communication tool but it is not a great knowledge base. We kept having important conversations and decisions made in Slack, but then it was very hard to go back and search through Slack's history to find threads that were key to internal and client projects.

File: _posts/2020-09-17-using-tettra.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+We also use Folders to keep our Draft pages before they are officially published to the whole team.
+
+<img src="/blog/assets/images/tettra/tettra_categories.jpg" alt="tettra categories" class="medium-img">
+
+## Adding Content
+
+Tettra allows you to create pages inside the tool as well as add a link to external documents (Google Doc or Google Sheet). This lets us create new pages as needed as well as link to our documents available through Google Docs and Google Sheets, keeping all the information in a single place.
+
+To make sure we keep things organized and standardized, we created a style guide for our Tettra pages that can be found under the Using Tettra category. Additionally, we also have a standard procedure for adding content.
```
Comment (etagwerker) [inline _posts/2020-09-17-using-tettra.md:32]: This is a little bit repetitive as you already mentioned the style guide in the Getting Started section. 😄 

Not a big deal though.

File: _posts/2020-09-17-using-tettra.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+Tettra allows users to either add new content in the form of a Tettra page, link to an external document (Google Doc or Google Sheet) or make a suggestion.
+
+When a team member wants to add content to the knowledge base, they can simply create a new page and save it to the Drafts folder of the category they want to add the page to. Once the page is published under the Drafts folder, they can use the Comments feature to request a review from other team members. Once the page has been reviewed and approved, it is moved to the proper Folder under the Category and made available to the team.
+
+However, sometimes a team member wants to see something in the knowledge base but is not sure what's the best way to add it. In that case, they can create a suggestion and either assign it to the best team member to write a page or to the knowledge base admin, who'll make sure the suggestion is addressed.
```
Comment (etagwerker) [inline _posts/2020-09-17-using-tettra.md:38]: @ABizzinotto In this paragraph it seems that we are missing a step. Maybe you can mention that when they want to find or see something they can easily search through all Tettra pages and create a suggestion for an existing page? I think the searchability of the KB is the most important feature. :)

File: _posts/2020-09-17-using-tettra.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+Through the integration we can also easily search for content. Tettra offers a variety of Slack commands that allow us to search for content, add a new page or create a new suggestion.
+
+Finally, the integration also adds the Tettra Wiki bot, which notifies a user in Slack when they are tagged in a Tettra page or suggestion.
+
+<img src="/blog/assets/images/tettra/slack-notification.jpg" alt="tettra Slack notification" class="medium-img">
+
+## Custom Integration
+
+When we thought of the knowledge base, it was important to us to connect it to our blog posts, as it contains a great deal of information on our procedures and how we use the tools we choose to use.
```
Comment (etagwerker) [inline _posts/2020-09-17-using-tettra.md:74]: @ABizzinotto You don't explicitly mention GitHub, but maybe you could mention it? Our blog is an OSS project in GitHub and we had to write a custom integration for the two services to work together. Also, maybe you can mention that one of our core values is "Open by Default" and we needed to be able to index our two blogs?

File: _posts/2020-09-17-using-tettra.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+## Keeping the Knowledge Base Up to Date
+
+Documentation becomes obsolete, policies change and so on. Adding content is a good way to make sure all the information is safely stored, but removing obsolete content and updating existing pages is also important to keep the knowledge base clean and relevant.
+
+In order to ensure our content is always up to date, we have a set process to review articles in the Knowledge Base every three months. We also get notifications from Tettra when a suggestion hasn't had any activity in a while, which is very helpful.
+
+## Conclusion
+
+In order to keep every team member on the same page and make information accessible to everyone in the easiest way possible, Tettra was our tool of choice. It integrates nicely with the tools we use, has an easy to use interface and allows us to keep the content updated and accessible.
```
Comment (etagwerker) [inline _posts/2020-09-17-using-tettra.md:86]: @ABizzinotto Not sure if it is worth it to mention the other tools that you evaluated? And why we picked Tettra over the others? IIRC it was related to the integrations they had and how they worked with our infrastructure/existing services (Google Drive + GitHub + Slack) ?

**Review by etagwerker — COMMENTED** [review]
@ABizzinotto Very thorough! I added a couple of comments. 👍

File: _posts/2020-09-17-using-tettra.md (hunk 6 above)
Comment (ABizzinotto) [reply _posts/2020-09-17-using-tettra.md:10]: @etagwerker the idea is for this article to be about the tool itself. The motivation and best practices on a knowledge base where shared on this other article here https://www.ombulabs.com/blog/team/agile/remote/organizing-team-knowledge-bases.html

So I didn't want to be repetitive here.

File: _posts/2020-09-17-using-tettra.md (hunk 7 above)
Comment (ABizzinotto) [reply _posts/2020-09-17-using-tettra.md:32]: This was on purpose :) I wanted to emphasize the style guide as a very common problem in knowledge bases is content being added in all sorts of different ways making it hard to read and update. Perhaps not the best way to go about it though...

File: _posts/2020-09-17-using-tettra.md (hunk 8 above)
Comment (ABizzinotto) [reply _posts/2020-09-17-using-tettra.md:38]: Ah! Great point 👍 I'll update it.

Comment (ABizzinotto) [reply _posts/2020-09-17-using-tettra.md:38]: Actually, I ended up adding a new section about searching for content :)

File: _posts/2020-09-17-using-tettra.md (hunk 9 above)
Comment (ABizzinotto) [reply _posts/2020-09-17-using-tettra.md:74]: Good idea 👍

File: _posts/2020-09-17-using-tettra.md (hunk 10 above)
Comment (ABizzinotto) [reply _posts/2020-09-17-using-tettra.md:86]: Hmm... I didn't want to name any other tools but I added a bit more info on why we went with Tettra and a shout out to their support (they were very helpful in the evaluation process).

**Review by etagwerker — APPROVED** [review]
@ABizzinotto Looks great! 💯

**Pull request merged**